SCClassLibrary/Common/Streams/Pspawner.sc

```diff
@@ -29,8 +29,8 @@ Spawner : Pattern {
 		^stream;
 	}
 
-	seq { | pat |
-		pat.embedInStream(event)
+	seq { | pattern |
```

Comment:
I changed this argument name to be consistent with the argument name in `par`. I wouldn't normally do this, but this class is undocumented, and it's unlikely anyone used the keyword argument syntax for this method in their existing code.